modules/sdk-coin-algo/src/algo.ts

```diff
@@ -529,6 +529,19 @@ export class Algo extends BaseCoin {
     return AlgoLib.algoUtils.publicKeyToAlgoAddress(pubKey);
   }
 
+  /**
+   * Gets config for how token enablements work for this coin
+   * @returns
+   *    requiresTokenEnablement: True if tokens need to be enabled for this coin
+   *    supportsMultipleTokenEnablements: True if multiple tokens can be enabled in one transaction
+   */
+  getTokenEnablementConfig() {
+    return {
+      requiresTokenEnablement: true,
```

Comment:
These definitions should be imported from statics

Reply:
Since this issue must be escalated to:

https://github.com/BitGo/BitGoJS/blob/c23c284c2edd91908012fcd22be73608c3f57847/modules/sdk-coin-sol/src/sol.ts#L919

https://github.com/BitGo/BitGoJS/blob/c23c284c2edd91908012fcd22be73608c3f57847/modules/sdk-coin-hbar/src/hbar.ts#L371

Can I create a ticket as tecnical debt ?

Reply:
I find a issue with algo, the sdk-coin-algo doesn't use static. 
here a first approach 
https://github.com/BitGo/BitGoJS/pull/3923

I note that static uses can be generalized with a  abstract StaticCoin that unify this getter and the orther (name, family, base factor, etc)

Do you want me to put something together?